e2e/assertions.js

```diff
@@ -0,0 +1,32 @@
+import {getTbtcAccountBalance} from "./common.js";
+
+export const assertMintedTbtcAmount = (web3, deposit, expectedTbtcAmount) => {
+    const actualTbtcAmountBn = web3.utils.toBN(deposit.tbtcAmount)
+    const expectedTbtcAmountBn = web3.utils.toBN(expectedTbtcAmount)
+
+    if (!actualTbtcAmountBn.eq(expectedTbtcAmountBn)) {
+        throw new Error(
+            `unexpected minted TBTC amount:
+                actual:   ${actualTbtcAmountBn}
+                expected: ${expectedTbtcAmountBn}`
+        )
+    }
+}
```

Comment:
Maybe we don't need to extract it to a separate function and do this check directly in test code. I was a little bit lost with the wrapper for the deposit we've defined for deposit object from tbtc.js.

Reply:
I think we will need to do the same in further tests hence I extracted this assertion.